modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -1,503 +1,302 @@
+[source, syntax]
 ----
-
-Assume for each abstract, you have generated a 1536-dimensional vector `embedding` of the abstract's `text` using Open AI's `text-embedding-ada-002` model.
-This model suggests a xref:indexes-vector-similarity-cosine[cosine similarity].
-For more information, see link:https://platform.openai.com/docs/guides/embeddings/which-distance-function-should-i-use[OpenAI's official documentation].
-
-You can create a cosine node vector index over the `embedding` property.
-
-.Query
-[source,cypher]
-----
-CREATE VECTOR INDEX `abstract-embeddings`
-FOR (n: Abstract) ON (n.embedding)
+FOR ()-[r:REL_TYPE]-() ON (r.embedding)
 OPTIONS {indexConfig: {
- `vector.dimensions`: 1536,
- `vector.similarity_function`: 'cosine'
+ `vector.dimensions`: $dimension,
+ `vector.similarity_function`: $similarityFunction
 }}
 ----
-====
 
-.Create a relationship vector index label:new[Introduced in 5.18]
-====
+== Query vector indexes
 
-Assume you have a graph of employees and their managers, who are themselves employees.
-Managers review their reports periodically, and you wish to search for reviews of similar themes and nuances to find excellent employees.
+To query a node vector index, use the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures/#procedure_db_index_vector_queryNodes[`db.index.vector.queryNodes`] procedure.
 
-.Data model
-[source,syntax,role="noheader",indent=0]
+.Signature for `db.index.vector.queryNodes`
+[source,syntax]
 ----
-(:Manager)-[:REVIEWED]->(:Employee)
+db.index.vector.queryNodes(indexName :: STRING, numberOfNearestNeighbours :: INTEGER, query :: ANY) :: (node :: NODE, score :: FLOAT)
 ----
 
-Assume for each review, you have generated a 256-dimensional vector `embedding` of the review's `text` using a shortening of Open AI's `text-embedding-3-large` model.
-This model suggests a xref:indexes-vector-similarity-cosine[cosine similarity].
-For more information, see link:https://platform.openai.com/docs/guides/embeddings/which-distance-function-should-i-use[OpenAI's official documentation].
-
-You can create a cosine relationship vector index over the `embedding` property.
+* The `indexName` refers to the unique name of the vector index to query.
+* The `numberOfNearestNeighbours` refers to the number of nearest neighbors to return as the neighborhood.
+* The `query` vector refers to the `LIST <INTEGER | FLOAT>` in which to search for the neighborhood.
 
-.Query
+.Find the 5 movies with the most similar plot to The Godfather
 [source,cypher]
 ----
-CREATE VECTOR INDEX `review-embeddings`
-FOR ()-[r:REVIEWED]-() ON (r.embedding)
-OPTIONS {indexConfig: {
- `vector.dimensions`: 256,
- `vector.similarity_function`: 'cosine'
-}}
+MATCH (m:Movie {title: 'Godfather, The'})
+CALL db.index.vector.queryNodes('moviePlots', 5, m.embedding)
+YIELD node, score
+RETURN node.title AS title, node.plot AS plot, score
```

Comment:
Using an alias for the returned `node` or `relationship` variable is good practice we want to encourage, especially for readability.
```suggestion
MATCH (m:Movie {title: 'Godfather, The'})
CALL db.index.vector.queryNodes('moviePlots', 5, m.embedding)
YIELD node AS movie, score
RETURN movie.title AS title, movie.plot AS plot, score
```
I've used `movie` here, but could quite easily have `m` be `movie` and use the alias `neighborMovie` or something if you really want, and dislike mixing single letter aliases with word aliases.